aleth/main.cpp

```diff
@@ -451,63 +452,43 @@ int main(int argc, char** argv)
         peers = vm["peers"].as<int>();
     if (vm.count("peer-stretch"))
         peerStretch = vm["peer-stretch"].as<int>();
+
     if (vm.count("peerset"))
     {
-        string peerset = vm["peerset"].as<string>();
-        if (peerset.empty())
+        string peersetStr = vm["peerset"].as<string>();
+        vector<string> peersetList;
+        boost::split(peersetList, peersetStr, boost::is_any_of(","));
```

Comment:
I think should be no need for `is_any_of`

Reply:
Actually, couldn't we instead use regex itself to get an array of matches in the `peersetStr`?

Reply:
> I think should be no need for `is_any_of`
split needs a predicate to determine if the current char is a delimiter...what did you have in mind instead of is_any_of?

> Actually, couldn't we instead use regex itself to get an array of matches in the `peersetStr`?
We could, but I think that's a bit overkill and would be more cumbersome since we'll need to use regex capture groups and iterate over the groups. I'd prefer to leave this code as-is.

Reply:
It was a guess about `is_any_of`, I thought maybe you can pass just a character delimiter to it, but looks like maybe you can't, sorry